### PR TITLE
Ruff: add and fix Q (except Q000)

### PR DESCRIPTION
**Pull request by kiblik**
Fix easier part of Q rules

https://docs.astral.sh/ruff/rules/#flake8-quotes-q

### CODE REVIEW

**Review by Maffooch — APPROVED** [review]

**Review by cneill — APPROVED** [review]

**Review by blakeaowens — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Comment by mtesauro** [comment]
@kiblik We're ready to merge this once the merge conflict is sorted out. :+1:

**Comment by mtesauro** [comment]
Going to go ahead and merge this since it's a MySQL test that is failing.

**Pull request merged**
